_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+* Use ``55.55.55.55` for my SPINNAKER_ENDPOINT in the below steps
+
+If you are using an existing NGINX ingress controller, or other services are likely to be using the same NGINX ingress controller, you should create a DNS entry that points at your NGINX ingress controller endpoint. (either a `CNAME Record` that points at the DNS name, or an `A Record` that points at the IP address).
+
+For example, if I get `abcd1234abcd1234abcd1234abcd1234-123456789.us-west-2.elb.amazonaws.com`, then I can do the following:
+* Create a CNAME pointing `spinnaker.domain.com` at `abcd1234abcd1234abcd1234abcd1234-123456789.us-west-2.elb.amazonaws.com`
+* Put `spinnaker.domain.com` in the `host` field in the below manifest (and uncomment it)
+* Use `spinnaker.domain.com` for my SPINNAKER_ENDPOINT in the below steps
+* (Alternately, for testing, create an `/etc/hosts` entry pointing spinnaker.domain.com at the IP address that `abcd1234abcd1234abcd1234abcd1234-123456789.us-west-2.elb.amazonaws.com` resolves to)
+
+If I get `55.55.55.55`, then I can do the following:
+* Create n A Record pointing `spinnaker.domain.com` at `55.55.55.55`
+* Put `spinnaker.domain.com` in the `host` field in the below manifest (and uncomment it)
+* Use `spinnaker.domain.com` for my SPINNAKER_ENDPOINT in the below steps
+* (Alternately, for testing, create an `/etc/hosts` entry pointing `spinnaker.domain.com` at `55.55.55.55`)
+
+Create a Kubernetes Ingress manifest to expose spin-deck and spin-gate (change your hosts and namespace accordingly):
+
+```bash
+tee spin-ingress.yaml <<-'EOF'
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: spin-ingress
+  labels:
+    app: spin
+    cluster: spin-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  rules:
+  - 
+    # host: spinnaker.some-url.com
+    # ^ If you have other things running in your cluster, you should uncomment this line and specify a valid DNS name
+    http:
+      paths:
+      - backend:
+          serviceName: spin-deck
+          servicePort: 9000
+        path: /
+      - backend:
+          serviceName: spin-gate
+          servicePort: 8084
+        path: /api/v1
+EOF
+```
+
+Create the Ingress
+
+```bash
+kubectl -n spinnaker apply -f spin-ingress.yaml
+```
+
+## Configure Spinnaker to be aware of its endpoints
+
+Spinnaker must be aware of its endpoints to work properly.
+
+This should be done from the halyard container:
+
+```bash
+SPINNAKER_ENDPOINT=http://spinnaker.domain.com
+# ^ Replace this with the IP address or DNS that points to your nginx ingress instance
+
+hal config security ui edit --override-base-url ${SPINNAKER_ENDPOINT}
+hal config security api edit --override-base-url ${SPINNAKER_ENDPOINT}/api/v1
+
+hal deploy apply
+```
+
+## Configuring TLS Certificates
+
+Configuration of TLS certificates for ingresses is often very environment-specific. In general, you would want to do the following:
+
+- Add certificate(s) so that your ingress controller can use them
```

Comment:
is there more information on how to do this?

Reply:
will do in v2 (once i figure out the collapsing)